docs/marionette.application.md

```diff
@@ -130,11 +130,11 @@ var App = Mn.Application.extend({
     console.log('Initialize');
   },
 
-  onBeforeStart: function(options) {
-    this.model = new MyModel(options.data);
+  onBeforeStart: function(app, options) {
+    this.model = new MyModel(this.options.data);
```

Comment:
Yep.. the real problem here is line 143.  `options` should be being passed to start and not at instantiation as mentioned in the note below it.